### PR TITLE
Update `AboutUs` page and Add `ppp` page

### LINKED ISSUE #2: Update `AboutUs` page
## Tasks
- Add your own details with suitable photo
    - Filename of photo should be `docs/images/github_username_in_lower_case.png`
    - Note the need for lowercase and `.png` format
- Indicate different roles and responsibilities played by each member

[Link](https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html#2-update-project-website-aboutus-readme) to details

### Team checklist
- [x] Jason #8 
- [x] Santosh #22 
- [x] Yun Ru #23 
- [x] Pep #11
- [x] Shenyi #20 

Note: Please add link to PR next to your name once done :)

### PR DESCRIPTION
**Pull request by ShenyiCui**
## Issues
- Addresses #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,22 @@
...
+3. Clone the project
+
+## Site Map
+- [User Guide](#)
+- [Developer Guide](#)
+- [About Us](#)
+- [Contact Us](#)
+
+## Acknowledgements
+- This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
```
Comment (jasonqiu212) [inline README.md:24]: is this `AboutUs` page supposed to be edited in this PR?

File: docs/AboutUs.md [hunk 2]
```diff
@@ -31,12 +31,12 @@ You can reach us at the email `jason.qiu@u.nus.edu`
 
 ### Johnny Doe
```
Comment (jasonqiu212) [inline docs/AboutUs.md:32]: dont forget to include your name haha

**Review by jasonqiu212 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ShenyiCui) [reply README.md:24]: Let me edit this in the other PR, i made a mistake and forgot to make a branch for this feature

File: docs/AboutUs.md (hunk 2 above)
Comment (ShenyiCui) [reply docs/AboutUs.md:32]: O shoot, good catch, thanks!

**Review by jasonqiu212 — APPROVED** [review]
lgtm!

**Pull request merged**